src/scales/scale.linearbase.js

```diff
@@ -27,6 +27,9 @@ module.exports = function(Chart) {
 				}
 			}
 
+			var setMin = tickOpts.min !== undefined || tickOpts.suggestedMin !== undefined;
```

Comment:
Good question. I'm not sure. The only way that could happen is if a user explicitly sets it (the default is undefined)